lighthouse-cli/chrome-launcher.ts

```diff
@@ -70,10 +70,16 @@ class ChromeLauncher {
   flags() {
     const flags = [
       `--remote-debugging-port=${this.port}`,
-      '--disable-extensions',
       '--disable-translate',
-      '--disable-default-apps',
-      '--no-first-run',
+      '--disable-extensions', // disable all chrome extensions entirely
+      // '--ignore-certificate-errors', // Ignore certificate errors, like self-signed
```

Comment:
Maybe we should make it a bit more general. Ignore ssl certs you probably only want it in dev mode. Maybe there are other options we want? We could expose it as `--dev` like we do with `--mobile`

it could underneath use `--chrome-flags="--ignore-certificate-errors"`